### PR TITLE
Test

### PR DESCRIPTION
**Pull request by Ashif-H**
Added Caching, Athena Querying Logic, README file!

### CODE REVIEW

File: app/Authenticate.py [hunk 1]
```diff
@@ -5,8 +5,12 @@
 
 load_dotenv()
```
Comment (pradish-pranam) [inline app/Authenticate.py:7]: Please remove the commented tenant_id.Id's should only be present in env

File: app/Authenticate.py [hunk 2]
```diff
@@ -5,8 +5,12 @@
...
 
+# tenant_id = 'fab3949d-3cb0-4097-802f-053ad7075219'
+
 def authenticate_access_token(access_token: str):
     try:
+
```
Comment (pradish-pranam) [inline app/Authenticate.py:12]: correct the function to actually check the token instead of just returning true

File: data/booker.parquet
Comment (pradish-pranam) [inline data/booker.parquet:1]: Never include customer data in a repo.Please remove!!

File: app/Functions.py [hunk 3]
```diff
@@ -10,6 +10,11 @@ async def limit_dict(original_dict, limit):
```
Comment (pradish-pranam) [inline app/Functions.py:10]: sort_months and sort_months_old does the same thing .Then why have both?

**Review by pradish-pranam — CHANGES_REQUESTED** [review]
Please correct the reviewed one's. Good work overall!

**Comment by Ashif-H** [comment]
Changes has been done! kindly check it.

**Pull request merged**